### ISSUE TITLE
Cross-Fade - possible bug?

### ISSUE THREAD

**Issue by f3lin3** [ififfy/flipflip#78]
When the next image to be shown is the same as the current image, and cross-fade images is selected then a warning is thrown at react-dom.development.js:506 in connection with duplicate keys:  

----
```
Warning: Encountered two children with the same key, `file:///home/cat1.jpg`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
    in div (created by ImagePlayer)
    in ImagePlayer (created by HeadlessScenePlayer)
    in div (created by HeadlessScenePlayer)
    in HeadlessScenePlayer (created by Player)
    in div (created by Player)
    in Player (created by Meta)
    in div (created by Meta)
    in Meta
```
------

This conflict happens less frequently the more images there are - so to quickly repro this - create a folder with only a one or two image files, set cross-fade on and run in the debugger to monitor the console logs.

As time goes on - more <div class="ImageView u-fill-container"> elements are created and appear orphaned in the ImagePlayer - consuming resources.

**Comment by ififfy** [comment]
Easy to fix:

* Add `ImagePlayer.state.nextImageID=0`
* Increment it whenever an image is ready to be added to `readyToDisplay`
* Set `img.key=this.state.nextImageID`
* Instead of `key={img.src]`, use `key={img.key}`

**Comment by GoddessServant0** [comment]
Fixed in #82 , thanks for your contribution @f3lin3 😁
